### PR TITLE
Create pipeline_refmap_yml.md

### PR DESCRIPTION
**Pull request by giuliaelgarcia**
Created the refmap yaml documentation

### CODE REVIEW

**Comment by deevdevil88** [comment]
@bio-la  can you  also eview the refmap one please after i am done . As i havent used this workflow a lot  and done refmapping outside panpipes, i might miss something.

File: docs/yaml_docs/pipeline_refmap_yml.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - <span class="parameter">threads_high</span> `Integer`, Default: 1<br>
+        Number of threads used for high intensity computing tasks. 
+        For each thread, there must be enough memory to load all your input files at once and create the MuData object.
+
+  - <span class="parameter">threads_medium</span> `Integer`, Default: 1<br>
+        Number of threads used for medium intensity computing tasks.
+        For each thread, there must be enough memory to load your mudata and do computationally light tasks.
+
+  - <span class="parameter">threads_low</span> `Integer`, Default: 1<br>
+  	    Number of threads used for low intensity computing tasks.
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_refmap_yml.md:41]: the space indent doesnt match the lines above. please change

**Review by deevdevil88 — CHANGES_REQUESTED** [review]
just one space indent change but other looks ok to me @giuliaelgarcia

File: docs/yaml_docs/pipeline_refmap_yml.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+- <span class="parameter">totalvi:</span> `String`, Default: path/to/totalvi<br>
+Provide path to totalvi saved model.
+
+- <span class="parameter">impute_proteins</span> `Boolean`, Default: True<br> CHEEEEECKKKKKK
+- <span class="parameter">transform_batch</span> `String`, Default:<br>
+Transform_batch is a batch-covariate specific to totalvi, allows the model to use the batch information in the query to mitigate 
+differences in protein sequencing depth.
+- <span class="parameter">scvi</span> `String`, Default:path/to/scvi<br>
+- <span class="parameter">sacnvi</span> `String`, Default:path/to/scanvi<br>
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_refmap_yml.md:79]: typo, should scanvi

File: docs/yaml_docs/pipeline_refmap_yml.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## Scvi tools parameters 
+
+- <span class="parameter">reference_data</span> `String`, Default: path/to/mudata<br>
+Specify one or more reference models to be used as reference. Users can also specify their own reference built using `pipeline_integration`.
+Leave blank for no model specification.
+
+- <span class="parameter">totalvi:</span> `String`, Default: path/to/totalvi<br>
+Provide path to totalvi saved model.
+
+- <span class="parameter">impute_proteins</span> `Boolean`, Default: True<br> CHEEEEECKKKKKK
```
Comment (deevdevil88) [inline docs/yaml_docs/pipeline_refmap_yml.md:74]: @bio-la  we werent sure if this is meant to True or False by default

**Review by deevdevil88 — CHANGES_REQUESTED** [review]
some typos need fixing

**Review by bio-la — APPROVED** [review]

**Review by bio-la — APPROVED** [review]

**Review by deevdevil88 — APPROVED** [review]

**Pull request merged**